### PR TITLE
updating globus guide

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

File: _uw-research-computing/globus.md [hunk 1]
```diff
@@ -24,40 +24,40 @@ own computer first.
...
-[`/staging`](file-avail-largedata.html) directory and requires login 
-access to the `transfer.chtc.wisc.edu` server.
-Therefore, to use Globus, you will need access to both of those 
-(instructions for requesting access are 
-[here](file-avail-largedata.html#1-policies-and-intended-use);
-mention that you want to use the Globus endpoint in your email).
-
-You will need to be able to
+All file transfer via Globus at CHTC requires: 
+- a group or user [`/staging`](file-avail-largedata.html) or `/projects` folder
```
Comment (aowen-uwmad) [inline _uw-research-computing/globus.md:28]:
```suggestion
- access to a directory in the [`/staging`](file-avail-largedata.html) or `/projects` folders
```

File: _uw-research-computing/globus.md [hunk 2]
```diff
@@ -24,40 +24,40 @@ own computer first.
...
-All file transfer via Globus at CHTC is based out of the
-[`/staging`](file-avail-largedata.html) directory and requires login 
-access to the `transfer.chtc.wisc.edu` server.
-Therefore, to use Globus, you will need access to both of those 
-(instructions for requesting access are 
-[here](file-avail-largedata.html#1-policies-and-intended-use);
-mention that you want to use the Globus endpoint in your email).
-
-You will need to be able to
+All file transfer via Globus at CHTC requires:
```
Comment (aowen-uwmad) [inline _uw-research-computing/globus.md:27]:
```suggestion
All file transfer via Globus at CHTC requires: 

```

File: _uw-research-computing/globus.md [hunk 3]
```diff
@@ -24,40 +24,40 @@ own computer first.
...
-Therefore, to use Globus, you will need access to both of those 
-(instructions for requesting access are 
-[here](file-avail-largedata.html#1-policies-and-intended-use);
-mention that you want to use the Globus endpoint in your email).
-
-You will need to be able to
+All file transfer via Globus at CHTC requires: 
+- a group or user [`/staging`](file-avail-largedata.html) or `/projects` folder
+- login access to the `transfer.chtc.wisc.edu` server. 
+Contact us at chtc@cs.wisc.edu if you need either of the above.
```
Comment (aowen-uwmad) [inline _uw-research-computing/globus.md:30]:
```suggestion

Contact us at chtc@cs.wisc.edu if you need either of the above. 
```

File: _uw-research-computing/globus.md [hunk 4]
```diff
@@ -24,40 +24,40 @@ own computer first.
...
-It can be found in the Globus web interface 
-[here](https://app.globus.org/file-manager/collections/d0bae6da-db3b-11ea-85a2-0e1702b77d41/overview?back=endpoints),
-or by searching endpoints for that name.
-If you  need the actual endpoint UUID, it is listed on the above page near the bottom
+The first step is to find the CHTC Globus endpoints. They can be found in the Globus web interface 
+by searching endpoints for "CHTC Staging" or "CHTC Projects".
+
+![CHTC Globus endpoints](../../images/guides-globus-endpoints.png)
+
+Or can be found at these links:
```
Comment (aowen-uwmad) [inline _uw-research-computing/globus.md:49]:
```suggestion
Or can be found at these links: 

```

**Review by aowen-uwmad — CHANGES_REQUESTED** [review]
Fixed some markdown formatting. Lists need to be separated from paragraph text by a blank line in order to be recognized as lists.

File: _uw-research-computing/globus.md [hunk 5]
```diff
@@ -24,40 +24,43 @@ own computer first.
...
-The first step is to find the CHTC Globus endpoint.
-The CHTC Globus endpoint's name is `chtc#staging`.
-It can be found in the Globus web interface 
-[here](https://app.globus.org/file-manager/collections/d0bae6da-db3b-11ea-85a2-0e1702b77d41/overview?back=endpoints),
-or by searching endpoints for that name.
-If you  need the actual endpoint UUID, it is listed on the above page near the bottom
+The first step is to find the CHTC Globus endpoints. They can be found in the Globus web interface 
+by searching endpoints for "CHTC Staging" or "CHTC Projects".
+
+![CHTC Globus endpoints](../../images/guides-globus-endpoints.png)
```
Comment (ChristinaLK) [inline _uw-research-computing/globus.md:49]:
```suggestion
![CHTC Globus endpoints](/images/guides-globus-endpoints.png)
```

**Review by aowen-uwmad — APPROVED** [review]

**Pull request merged**